navigation/scripts/nav.py

```diff
@@ -28,7 +29,9 @@ def __init__(self):
         self.robot_pose = Pose()
         s = rospy.Service('navigate_to', NavigateTo, self.setNavigate)
 
-        # sub
+        # pub/sub
+        self.nav_done_pub = rospy.Publisher('nav_done', Bool, queue_size=10)
```

Comment:
change the nav service to just a publisher no? 
Regards,
Camenbert fromage

Reply:
this is a topic for notifying the rest of the system that the navigation has completed, I think it makes sense as is?